OTAnalytics/domain/flow.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from OTAnalytics.domain.section import Section
+
+
+@dataclass(frozen=True)
+class FlowId:
+    id: str
+
+    def serialize(self) -> str:
+        return self.id
+
+
+class Flow:
+    def __init__(
+        self,
+        id: FlowId,
+        start: Section,
+        end: Section,
```

Comment:
Should we prevent passing something other than a Section?

We could also think about prevent passing the same section as both start and end (but I also can think of a use case of u-turns, so maybe this is not necessary)